src/usr/bin/gce-nic-naming

```diff
@@ -42,22 +42,16 @@ declare -i accelerator_to_ethernet_ratio=1
 logger -t "gce-nic-naming" -p local0.info "Run ${LOG_TAG}"
 
 ###############################
-# Log error and exit with -1
+# Log error and exit with 1
 # Environment:
-#   ID_NET_NAME_PATH: net_id name based off of PCI path in form enp##s#
```

Comment:
why are we removing the use of this env var?

Reply:
This was the env var that udev uses to communicate what the "predictable" name is. We shouldn't be using it because we do not know whether the host has configured predictable names or not, we should be deferring to udev.